.github/workflows/auto-update.js

```diff
@@ -44,6 +41,31 @@ module.exports = async ({ github, context, core, require }) => {
     });
   };
 
+  function update_flake() {
+    return new Promise((resolve, reject) => {
+      exec(`nix flake update`, (error, stdout, stderr) => {
+        if (error) {
+          return reject(error);
+        }
+
+        const shas = [
+          ...stderr.matchAll(
+            /NixOS\/nixpkgs\/(.*)' \(([0-9]{4}-[0-9]{2}-[0-9]{2})\)/g
```

Comment:
Case insensitive because `nixos/nixpkgs` certainly also works.
```suggestion
            /NixOS\/nixpkgs\/(.*)' \(([0-9]{4}-[0-9]{2}-[0-9]{2})\)/gi
```